README.md

```diff
@@ -27,7 +26,7 @@ Currently the following platforms are supported:
 
 ### Prerequisites
 This package requires the TileDB shared library be installed and on the system path. See the
-[official TileDB installation instructions](https://docs.tiledb.com/main/how-to/installation)
+[official TileDB installation instructions](docs/building-from-source.md)
```

Comment:
This is for TileDB core, so I think we can just link to the docs recently added there

```suggestion
[TileDB GitHub repository](https://github.com/TileDB-Inc/TileDB/blob/main/BUILDING_FROM_SOURCE.md)
```